e2e/libs/longhorn_deploy/base.py

```
@@ -32,24 +32,42 @@ def check_longhorn_uninstall_pod_log(self):
-            raise
-        except subprocess.TimeoutExpired as e:
-            logging(f"Command timed out after {e.timeout} seconds")
-            logging(f"stdout: {e.output}")
-            raise
+    def create_longhorn_namespace(self):
+        command = "./pipelines/utilities/create_longhorn_namespace.sh"
+        process = subprocess.Popen([command, "create_longhorn_namespace"],
+                                   shell=False)
+        process.wait()
+        if process.returncode != 0:
+            logging(f"Creating longhorn namespace failed")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)